test/test_RamsesBlenderExporter.py

```diff
@@ -45,3 +47,159 @@ def test_default_scene_viewer_does_not_crash_blender(self):
             if not exportable_scene.is_valid():
                 validation_report = exportable_scene.get_validation_report()
                 raise RuntimeError(validation_report)
+
+    def test_scaling_x_by_2(self):
+        bpy.ops.wm.open_mainfile(filepath='../test_scenes/cube_scaledX_2.blend')
```

Comment:
Wonder if you noticed that the other tests pass the input file as a command line parameter to the script? The reason I did that is that you can invoke the same tests with different input files, and the test configuration (input file, expected screenshot etc.) are outside of the test code. Also, if you use relative paths like this, the tests become quite difficult to execute on a different machine/environment. Just re-use the base class for the tests and use the same mechanics :)

Reply:
Makes sense! I will try this approach then.